testproject/tests/test_second_step_authentication.py

```diff
@@ -198,14 +228,15 @@ def test_activation_otp_confirm_wrong(active_user):
         path="/auth/email/activate/",
         format="json",
     )
+    assert response.status_code == 200
     response = client.post(
         path="/auth/email/activate/confirm/",
         data={"code": "test00"},
         format="json",
     )
     assert response.status_code == 400
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too